lib/ansible/modules/crypto/openssl_certificate.py

```diff
@@ -58,9 +58,10 @@
         description:
             - Name of the provider to use to generate/retrieve the OpenSSL certificate.
             - The C(assertonly) provider will not generate files and fail if the certificate file is missing.
+            - The C(entrust) provider was added for Ansible 2.9 and requires credentials for the Entrust Certificate Services (ECS) API
```

Comment:
```suggestion
            - "The C(entrust) provider was added for Ansible 2.9 and requires credentials for the
               L(https://www.entrustdatacard.com/products/categories/ssl-certificates,Entrust Certificate Services) (ECS) API."
```
Or if you have a better URL, use that one :)

Reply:
> One thing you'll also need is a [changelog fragment](https://docs.ansible.com/ansible/latest/community/development_process.html#changelogs-how-to) (with `minor_changes`).

Will do - should I also add a line to BOTMETA.yml for module_util maintenance, or is that something the ansible team maintains/is committed after the PR?
```
$module_utils/ecs:
    maintainers: team_ecs

team_ecs: ctrufan
```

Reply:
It's probably a good idea to add this to this PR as well. Please don't add a new team, just use
```.yaml
$module_utils/ecs/:
  maintainers: ctrufan $team_crypto
```
for now. Also feel free to add yourself as a maintainer for `openssl_certificate`, then you will be informed on issues/PRs regarding the module. Might be useful is something would affect the `entrust` provider.
```.yaml
$modules/crypto/openssl_certificate.py:
  maintainers: ctrufan
```
(The other maintainers are inherited from $modules/crypto and from the module author's list.)

Reply:
Sounds good - I wasn't sure if it was expected to use a new team, and I didn't know about the inheritance.